blueocean-dashboard/src/main/less/core.less

```diff
@@ -390,9 +390,11 @@ table.highlight {
     width: 100%;
     height: 100%;
     display: block;
-    padding: 0.5em 0;
+    padding: 0.5em 0.8em 0.5em 0;
     text-decoration: none;
     color: inherit;
+    line-height: 22px; // force height to fill space to ensure clicks are "captured"
+    .text-overflow;
```

Comment:
TBH I'd rather the overflow stuff be attached to another class that is added by default by our `<Table>` component, with a switch to exclude that class. That way when somebody wants wrapping on one column, they're not going to ditch the entire thing. We also might want to separate the no-wrap from the text-overflow, because if I have one column that's tall because of icons or something, you might as well wrap a long line once or twice before cutting it off with ellipses.